app/controllers/home_controller.rb

```diff
@@ -13,13 +13,10 @@ def index
   end
 
   def landing
-    recipe_images = RecipeImage.find([549, 376, 529, 526])
```

Comment:
I picked those recipes specifically because they had a decent number of revisions, forks, pictures, etc. We should probably move the magic numbers into a config file so it can be different in dev / prod, but just grabbing the last 5 in the database probably isn't what we want.